install

```diff
@@ -90,7 +92,7 @@ fi
 curl --fail --location --progress-bar --output "$tmp_tar" "$dune_tar_uri" ||
     error "Failed to download dune tar from \"$dune_tar_uri\""
 
-tar -xf "$tmp_tar" -C "$tmp_dir" 2>&1 > /dev/null ||
```

Comment:
If the initial intent was to completely hide output, then my change is correct.
If the intent was to hide only stdout, but put the stderr onto stdout, then it is not

Reply:
Given I removed the `-v` flag in a previous PR, normally there shouldn't be any output on stdout anyway.

Reply:
All good :+1: